cfgov/unprocessed/js/modules/UStreamPlayer.js

```diff
@@ -1,14 +1,14 @@
-'use strict';
+// TODO: Remove VideoPlayer module.
```

Comment:
Change this to UStreamPlayer module, the VideoPlayer is still used with YouTube.

Reply:
Oops, fixed